src/server/ua_subscription.h

```diff
@@ -18,6 +18,8 @@
 #include "ua_types.h"
 #include "ua_types_generated.h"
 #include "ua_session.h"
+#include "ua_workqueue.h"
+#include "ua_plugin_nodestore.h"
```

Comment:
Are these includes really necessary?

Reply:
The problem comes up after running clang-format, since the includes get rearranged (sorted by alphabet) and then the build breaks because these two includes were already done by a previous include which is now subsequent and therefore the include is not done yet and the symbols cannot be found.

So if i remember correctly, something like this is the case:
#include "zzz.h" // includes ua_workqueue.h and ua_plugin_nodestore.h
#include "ua_subscription.h"

After rearranging, zzz is after ua_subscription and therefore the workqueue references, which are used in the ua_subscription.h, cannot be found.
(sorry for the weird description, i hope you understand what I mean 😁)